library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -298,6 +340,26 @@ def target_root
 
         Yast::Installation.destdir
       end
+
+      def installation_helper_step_4
```

Comment:
is possible to use better name? we should try to abstract it a bit.

Reply:
Well, this is literally calling the step 4 of Snapper... which doesn't have a meaningful name on its own.